### PR TITLE
Signin#5

### PR DESCRIPTION
**Pull request by rahulcs754**
- added Signup 
- Use API call for login
- store login details in local storage and context
- add header logout button

image
image
image

### CODE REVIEW

File: src/Pages/Login/Components/Login.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  const submitHandler = async (e) => {
+    e.preventDefault();
+    console.log("jana hai");
+    try {
+      const response = await axios.post(`/api/auth/login`, {
+        email,
+        password,
+      });
+      console.log("ane do");
+      console.log(response);
```
Comment (kjais1720) [inline src/Pages/Login/Components/Login.js]: remove the console.logs

File: src/Pages/Login/Components/Login.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+      if (response.status === 200 || response.status === 201) {
+        console.log(response);
```
Comment (kjais1720) [inline src/Pages/Login/Components/Login.js]: remove it

**Review by kjais1720 — APPROVED** [review]
Everything looks good, just a small suggestion, export the components through index.js files, so that their import links look cleaner

File: src/Pages/Login/Components/Login.module.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
+/* Login css */
+.login_box {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (nishasen) [inline src/Pages/Login/Components/Login.module.css]: you can use utility class here

**Review by nishasen — APPROVED** [review]
code looks clean and good.

File: src/Pages/Login/Components/Login.js (hunk 1 above)
Comment (rahulcs754) [reply src/Pages/Login/Components/Login.js]: Thanks for pointing it out, fixed it in the latest commit

File: src/Pages/Login/Components/Login.js (hunk 2 above)
Comment (rahulcs754) [reply src/Pages/Login/Components/Login.js]: Thanks for pointing it out, fixed it in the latest commit

**Pull request merged**